### PR TITLE
Add alternating "idea" backrounds

### PR DESCRIPTION
**Pull request by felixbutton**
Adds alternating "idea" backrounds for better readability, especially on mobile devices where users could lose track on which img and which heading belong together.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -123,6 +123,15 @@
   display: grid;
   grid-template-rows: repeat(2, 1fr);
   grid-template-columns: 1fr;
+  background-color: white;
```
Comment (martinweck) [inline css/index.css]: maybe, we should define a variable for the colour `white`?

File: css/index.css [hunk 2]
```diff
@@ -163,6 +172,8 @@
     grid-template-columns: repeat(2, 1fr);
     grid-template-rows: 1fr;
     gap: 2rem;
+	min-height: auto;
```
Comment (martinweck) [inline css/index.css]: check the indent, please

File: css/index.css [hunk 3]
```diff
@@ -193,7 +204,7 @@
   }
 
   .idea__img-container.idea__img--large img {
-    width: 30rem;
+    width: 28rem;
```
Comment (martinweck) [inline css/index.css]: I thought about that, too. I think we can keep it with 30rem. Then it does not fit. It's exactly what we want to tell: The system grows until it is too large => it does not fit anymore.

File: css/index.css (hunk 2 above)
Comment (felixbutton) [reply css/index.css]: resolved in https://github.com/innoq/SCS/pull/64/commits/badf6fd2fab28a72921e0b2cc74c25e9c6d72324

File: css/index.css (hunk 1 above)
Comment (felixbutton) [reply css/index.css]: resolved in https://github.com/innoq/SCS/pull/64/commits/c27c5e26a1d6db629d3c6579b916caeda7985261

File: css/index.css (hunk 3 above)
Comment (felixbutton) [reply css/index.css]: reverted in https://github.com/innoq/SCS/pull/64/commits/2eb966d7095d45d894bff5b881cac521c3c6b5ac

**Pull request merged**